en/docs/chapter_hashing/hash_map.md

```diff
@@ -1,30 +1,30 @@
 # Hash table
 
-A <u>hash table</u> achieves efficient element querying by establishing a mapping between keys and values. Specifically, when we input a `key` into the hash table, we can retrieve the corresponding `value` in $O(1)$ time.
+A <u>hash table</u>, also known as a <u>hash map</u>, is a data structure that establishes a mapping between keys and values, enabling efficient element retrieval. Specifically, when we input a `key` into the hash table, we can retrive the corresponding `value` in $O(1)$ time complexity.
 
-As shown in the figure below, given $n$ students, each with two pieces of data: "name" and "student number". If we want to implement a query feature that returns the corresponding name when given a student number, we can use the hash table shown in the figure below.
+As shown in the diagram below, given $n$ students, each student has two data fields: "Name" and "Student ID". If we want to implement a query function that takes a student ID as input and returns the corresponding name, we can use the hash table depicted below.
```

Comment:
the hash table depicted below -> the hash table shown in the figure below

The keywords "the figure below" will be replaced with the Figure number during the build process, so don't modify them.

Reply:
done